### PR TITLE
Calendar Of Events: WGAG 2.1 & 2.2 AA Assessment

### PR DESCRIPTION
**Pull request by BrahimMahadi**
### This pull request includes the WCAG 2.1 & 2.2 Accessibility assessment done for the calendar of events plugin

General checklist
- [x] Updated calendar of events documentation to include the accessibility assessment reports
- [x] Plugin was tested against WCAG for accessibility
- [x] Documentation is bilingual

### CODE REVIEW

File: site/pages/docs/ref/cal-events/cal-events-en.hbs [hunk 1]
```diff
@@ -29,6 +29,14 @@
 	</ul>
 </section>
 
+<section>
+	<h2>Evaluation and report</h2>
+	<ul>
+		<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">English Report</a></li>
+		<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">French Report</a></li>
```
Comment (duboisp) [inline site/pages/docs/ref/cal-events/cal-events-en.hbs:36]:
```suggestion
		<li>Accessibility (SC WCAG 2.1/2.2 Level A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">English Report</a></li>
		<li>Accessibility (SC WCAG 2.1/2.2 Level A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">French Report</a></li>
```

File: site/pages/docs/ref/cal-events/cal-events-fr.hbs [hunk 2]
```diff
@@ -29,6 +29,14 @@
 	</ul>
 </section>
 
+<section>
+	<h2>Évaluation et rapport</h2>
+	<ul>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">Rapport Anglais</a></li>
+		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">Rapport Français</a></li>
```
Comment (duboisp) [inline site/pages/docs/ref/cal-events/cal-events-fr.hbs:36]:
```suggestion
		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">Rapport Anglais</a></li>
		<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">Rapport Français</a></li>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
@BrahimMahadi please update your assessment according to the new methodologies and the revised vocabulary terms

File: site/pages/docs/ref/cal-events/cal-events-fr.hbs (hunk 2 above)
Comment (BrahimMahadi) [reply site/pages/docs/ref/cal-events/cal-events-fr.hbs]: Fixed in commit "Updated assessment according to new methodologies"

File: site/pages/docs/ref/cal-events/cal-events-en.hbs (hunk 1 above)
Comment (BrahimMahadi) [reply site/pages/docs/ref/cal-events/cal-events-en.hbs]: Fixed in commit "Updated assessment according to new methodologies"

**Review by hanseller — CHANGES_REQUESTED** [review]
Please rename all JSON properties

**Comment by hanseller** [comment]
All changes were completed and committed to my Fork via branch PR9500

**Review by duboisp — APPROVED** [review]
Review and tested locally. All good

File: src/plugins/cal-events/reports/a11y-1.json [hunk 3]
```diff
@@ -0,0 +1,905 @@
...
+      "@type": ["wb-arc:Revision", "foaf:Person"],
+      "earl:mode": "earl:manual",
+      "earl:outcome": "earl:untested",
+      "outcome": "Untested",
+      "notes": "",
+      "date": "2022-09-22"
+    }
+  ],
+
+  "results": [
```
[the remark this answers is not in the document]
Comment (duboisp) [reply src/plugins/cal-events/reports/a11y-1.json]: fixed

**Pull request merged**